aip/cloud/2602.md

```diff
@@ -0,0 +1,122 @@
+---
+aip:
+  id: 2602
+  scope: cloud
+  state: approved
+  created: 2019-06-12
+  updated: 2019-06-12
+permalink: /cloud/2602
+redirect_from:
+  - /2602
+---
+
+# Commands taking different, non-overlapping arguments depending on a parameter
```

Comment:
In the interest of shortening some of these headings, maybe call this "Parameter-dependent arguments"?

Reply:
``suggestion
# Parameter-dependent arguments
```